src/app/(pages)/clients/components/ClientsTable.tsx

```diff
@@ -73,11 +73,16 @@ type ClientsTableProps = { count?: string; page?: string; me?: boolean };
 
 export default async function ClientsTable(props: Readonly<ClientsTableProps>) {
   const { count, page, me } = props;
-  const itemsPerPage = count ? parseInt(count) : 10;
-  const currentPage = page ? parseInt(page) + 1 : 1;
+  let itemsPerPage = 10;
+  let currentPage = 0;
+
+  itemsPerPage = count ? parseInt(count) || itemsPerPage : itemsPerPage;
+  currentPage = page ? parseInt(page) - 1 || currentPage : currentPage;
+
+  const startIndex = currentPage * itemsPerPage + 1;
   const response = me
-    ? await getMeClientsPage(itemsPerPage, currentPage)
-    : await getClientsPage(itemsPerPage, currentPage);
+    ? await getMeClientsPage(itemsPerPage, startIndex)
+    : await getClientsPage(itemsPerPage, startIndex);
```

Comment:
if user receives an error access denied, the client table component crashes

Reply:
Fixed in https://github.com/indigo-iam/iam-dashboard/pull/44/commits/2ade16d05eefe1d2a2c6ae6feee7319406bbe784